### ISSUE TITLE
Looks of site

### ISSUE THREAD

**Issue by Vinaydgr8** [fossevents/pychill#4]
Guys the contrast on the site is not looking good.
Also the background is repeating itself, so I suggest that we make a rolling text type website.Since I do not know css and xml I can't help it.

**Comment by c4ndyfl1p** [comment]
yaa i get that. so i did write another code with a different background and
a bunch of other stuff but idk why its not pushing it. it ws working
perfectly fine before! now it seems i dont have the authority 2 do
that......

**Comment by Ambar-H** [comment]
Guys ive created the website

link: https://www.pychill.comoj.com

I want to upload it on the main server because i have like 20 to 30 files which i cant just copy paste.

So i just wanted to know whether this pychill.fossevents.in has independant server or is on GITHUB ???

**Comment by theskumar** [comment]
@ambar-127 the website lives on Github itself. Whatever you add to `gh-pages` branch becomes live. You need to send a pull request to this Github repository after forking it. See: https://help.github.com/articles/using-pull-requests/. If you don't understand, let us know where are get stuck.

**Comment by theskumar** [comment]
@craziepixie I've made little adjustments to the styles. See: https://pychill.fossevents.in 
The way to update the code is to send another pull request with your changes. Once we review it, we will merge your changes to this repository.

**Comment by c4ndyfl1p** [comment]
yep.. its looking better!

**Comment by c4ndyfl1p** [comment]
ambar ur link's broken! tera code dekhna hai mujhe...update it

**Comment by bhanuvrat** [comment]
@craziepixie  if you want people to be notified about your messages mention them with an @ symbol ... like -> @ambar-127

**Comment by c4ndyfl1p** [comment]
oka thnx 

@ambar-127 bhai link sahi kar de!

**Comment by nileshgulia1** [comment]
hello @theskumar  can u tell me the mockup of the website.I may redesign it if u want...

**Comment by CuriousLearner** [comment]
@nileshgulia1 You can create a mockup as you like and share here with the community to finalize it :)

**Comment by nileshgulia1** [comment]
@CuriousLearner  I"ll send the mockup soon... :)

**Comment by nileshgulia1** [comment]
@CuriousLearner  here's the mockup please review:
![screenshot from 2017-11-26 20-18-29](https://user-images.githubusercontent.com/22280901/33241109-251f6dfa-d2e7-11e7-88db-10ab1d5cfbe1.png)

**Comment by CuriousLearner** [comment]
@nileshgulia1 I think that is actually the clean-blog theme.

Also, we're talking about PyChill website ( https://pychill.fossevents.in) here, not the FOSS Events one. So probably, you'd like to look on how it looks like currently.

**Comment by nileshgulia1** [comment]
@CuriousLearner  yeah I thought It will be good if we can make it simple.fine,  i will suggest some simple styling soon...:)

**Comment by nileshgulia1** [comment]
@CuriousLearner  here's the new mockup:
(the background is dynamic)
![screenshot from 2017-11-27 19-08-27](https://user-images.githubusercontent.com/22280901/33269465-d1eee6ae-d3a6-11e7-8130-c3e851520407.png)

**Comment by satyaakam** [comment]
Hi,

Can you think of a theme with Alphabets and kids etc... i like lighter themes with flat designs.

cheers
-Satya

**Comment by nileshgulia1** [comment]
hello @satyaakam  how about a theme with simple styling, may be we can use jekyll ,or any other bootstrap theme .here's one of the initialised theme:

![screenshot from 2017-11-27 20-26-35](https://user-images.githubusercontent.com/22280901/33272859-8555395a-d3b1-11e7-9a5e-6e4a87bd04e3.png)
